src/components/skills-quiz/SearchCourseCard.jsx

```diff
@@ -21,9 +21,18 @@ import { useSelectedSkillsAndJobSkills } from './data/hooks';
 import { useDefaultSearchFilters } from '../search/data/hooks';
 import { UserSubsidyContext } from '../enterprise-user-subsidy';
 
-const getCourseSkills = (course) => (
-  course.skill_names?.length > 0 ? course.skill_names.slice(0, MAX_VISIBLE_SKILLS_CARD) : null
-);
+const getCourseSkills = (course, selectedJobSkills) => {
+  const skillsArray = [];
+  if (course.skill_names?.length > 0) {
+    course.skill_names.map(skill => {
```

Comment:
can we replace map with filter, because filter fits best for this use case.
Something like
const commonSkills = course.skill_names.filter(skill => selectedJobSkills.includes(skill))

Reply:
done